backend/users-service/app/prisma/schema.prisma

```diff
@@ -0,0 +1,31 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "mongodb"
+  url      = env("DATABASE_URL")
+}
+
+model Question {
+  id          String   @id @default(auto()) @map("_id") @db.ObjectId
+  category    String
+  description String
+  difficulty  String
+  id_         String   @map("id")
+  languages   String[]
+  status      String
+  title       String
+  topic_tags  String[]
+  url         String
+  userId      String   @db.ObjectId
+  user        User     @relation(fields: [userId], references: [id])
+}
+
+model User {
+  id        String @id @default(auto()) @map("_id") @db.ObjectId
+  avatarUrl String
+  email     String
+  role      String
```

Comment:
is your intent for leaving `role` as a `String` is so that the restriction is enforced on an app level but not db level? (this makes it easier for db migrations)

Reply:
Actually, I just didn't know Prisma could do enums... changed `role` to an `enum` in e1b6b913c01d2eb06e1d4398c4b2192c5d1d00c4